lib/manageiq/providers/openstack/legacy/openstack_handle/handle.rb

```diff
@@ -380,7 +382,7 @@ def accessible_tenant_names
     def default_tenant_name
       return @default_tenant_name ||= "admin" if tenant_accessible?("admin")
       tenant_names.each do |name|
-        next if name == "services"
+        next if name.blank? || name == "services"
```

Comment:
Is there an example for case when name.blannk? == true? I've left it in example code from experimenting with tenant with empty name, but I haven't found such case in OSP.

Reply:
I didn't think so, but I thought perhaps you had a good reason for it. I'll pull the change if you agree it's unlikely to come up.

Reply:
Yep, I'd suggest drop it in that case.